src/transformers/models/gpt2/modeling_gpt2.py

```diff
@@ -484,7 +484,7 @@ def _init_weights(self, module):
         #
         # Reference (Megatron-LM): https://github.com/NVIDIA/Megatron-LM/blob/main/megatron/model/gpt_model.py
         for name, p in module.named_parameters():
-            if "c_proj" in name and "weight" in name:
+            if name == 'c_proj.weight':
```

Comment:
This should fix the code quality tests :)
```suggestion
            if name == "c_proj.weight":
```

Reply:
Ty @lvwerra . I should have run `make style` myself, I saw it in the docs but I thought there was no possible way I could have introduced an issue on a single line of code that simple, but here we are :). I read a style guide suggestion a long time ago to use single quotes for string literals (like here) and double quotes for English text for display to humans, and thought it was a reasonable rule and have used that distinction in my code since. Anyway, thanks!